securesignals-ima-devapp/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <application
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:supportsRtl="true"
+        android:theme="@style/Theme.UID2AndroidSDK">
```

Comment:
We don't need a Theme, given it's a Dev App.

Reply:
Done.